Resources/Textures/Objects/Weapons/Melee/claymore.rsi/meta.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from tgstation and modified by Taral at https://github.com/tgstation/tgstation/pull/31615/commits/1ffa93a49df1e728a4f55d6b867879d29a8bec1a",
```

Comment:
link didn't work for me, should it be https://github.com/tgstation/tgstation/pull/31615/commits/8795484578b33a9919b7801667287c0f28afe642?

Reply:
or https://github.com/tgstation/tgstation/pull/31400/commits/6046fb493bf487a679a235acfebfe3674cf2db8a?

Reply:
everything else is good btw

Reply:
ok. I fixed the commit from the meta.json